repository.rosinstall

```diff
@@ -31,10 +31,14 @@
 - git:
     local-name: lma_kinematics_plugin
     uri: 'https://github.com/shadow-robot/lma_kinematics_plugin'
-    version: F_ik_with_joint_limits
+    version: F_fixing_kinetic_build
+
+- git:
+    local-name: trac_ik
```

Comment:
But kinetic packages only appear in xenial and later Ubuntu. If you chek on Trusty you won't find them.
It seems that trac-ik is released on kinetic see https://github.com/ros/rosdistro/blob/master/kinetic/distribution.yaml#L6280